server/routes.go

```diff
@@ -892,6 +893,13 @@ func (s *Server) GenerateRoutes() http.Handler {
 }
 
 func Serve(ln net.Listener) error {
+	if debug := os.Getenv("OLLAMA_DEBUG"); debug != "" {
```

Comment:
This should be declared globally in cmd so it configures non-server commands too

Reply:
My intent was to only impact the server to narrow the scope.